.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+name: test jest
+
+on:
+  push: # se ejecuto el action cada que se hace push a las ramas espesificadas, no es necesario a mas
+    branches:
+      - main
+      - develop 
```

Comment:
**chore** 🧹
We are not going to use develop branch
```suggestion

```

<sub><sup>I'm using [conventional comments](https://conventionalcomments.org/) formatting to improve comments readability and reader's expectations in this PR.</sup></sub>